packages/web-react/src/components/ScrollView/README.md

```diff
@@ -0,0 +1,48 @@
+# ScrollView
+
+## Usage
+
+### Default (Horizontal)
+
+We need to set the maximum height if it is not defined by the parent.
+
+```javascript
+<div style={{ height: '250px' }}>
```

Comment:
I think it is mainly confusing because there is a wrong title here. Also, we don't need a wrapper. Because there are several ways to limit the height, it could be fixed height like this or it could be in some kind of grid layout etc. This should be explained that we set inline height only to showcase the ScrollView.

Reply:
With small help from our AI colleagues
> It's crucial to understand that the ScrollView component, like any HTML element, expands to fit its content. Without a height limit on the parent container, the ScrollView will stretch to include all its children, which could negate the need for scrolling.

> To make scrolling and scroll indicators function correctly, the parent container's height must be limited. In our examples, we set this height limit using inline styles for demonstration purposes only.

Reply:
done in README